packages/core/test/integration/fs/cleanup-file-system.ts

```diff
@@ -0,0 +1,55 @@
+import fs from 'fs';
```

Comment:
Why is this class needed? Removing a directory and it's children can simply be done using:

```ts
import fs from 'fs';

await fs.promises.rm('path/to/dir', { recursive: true });
```

See https://nodejs.org/api/fs.html#fspromisesrmpath-options

The reason to keep track of the concurrent IO tasks in the production code is that Stryker does so many parallel IO actions you can run out of OS-level file handles. We don't worry about that during integration testing.